### PR TITLE
Fixes #147 Implement lightbox feature for images

### LINKED ISSUE #147: Implement Lightbox for the images.
Currently, susper only shows images (with no URL linked). To enhance user experience, we should implement the light box technology similar to Google. _Clicking on the images, should open up in a lightbox with an image slider._

### PR DESCRIPTION
**Pull request by harshit98**
Fixes #147 Just to get started with the lightbox feature. I'll open an issue regarding, add more features to it like an image slider similar to Google's lightbox.

Screenshot -

screenshot from 2017-04-14 22-10-00

### CODE REVIEW

**Comment by harshit98** [comment]
@isuruAb Yeah, working on it. It's totally a different error. Even after adding `providers` it is showing the same error.

**Comment by mariobehling** [comment]
@harshit98 Thanks. Please provide a link to a sample set up on your GitHub pages.

**Comment by harshit98** [comment]
@mariobehling Yes, I'll provide you the preview link. I'm figuring it out. Please don't merge this PR until I provide you a preview link.

**Comment by harshit98** [comment]
@mariobehling Live preview <a href="https://harshit98.github.io/susper.com/">here</a>. If there is some problem let me know. :smile:

**Comment by isuruAb** [comment]
@harshit98  it is not working .it does not give any search result. can you check it

**Comment by harshit98** [comment]
@isuruAb ok. I'll look, I don't know why it is not working.

**Comment by isuruAb** [comment]
@harshit98 here is the console log
`vendor.ed0ad41….bundle.js:1343 Mixed Content: The page at 'https://harshit98.github.io/susper.com/search?query=h&verify=false&nav=file…=5&resource=global&prefermaskfilter=&maximumRecords=10&timezoneOffset=-330' was loaded over HTTPS, but requested an insecure script 'http://yacy.searchlab.eu/solr/select?query=h&verify=false&nav=filetype,prot…`

HTTPS is the problem

File: src/app/app.component.html [hunk 1]
```diff
@@ -1 +1,2 @@
 <router-outlet></router-outlet>
+<ngbox></ngbox>
```
Comment (daminisatya) [inline src/app/app.component.html]: CRLF

**Pull request closed without merging**